src/components/paginationbuttons/PaginationButtons.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import './PaginationButtons.css';
+
+function PaginationButtons({ onPrev, onNext, disablePrev, disableNext }) {
+    return (
+        //reusable component for the two navigation buttons
+        <div>
+            <button onClick={onPrev} disabled={disablePrev}>
```

Comment:
Geef je buttons altijd netjes een type mee (in dit geval type="button") want submit is de default als je niks meegeeft